tests/.suitespec.json

```diff
@@ -26,18 +26,785 @@
+        ],
+        "pymysql": [
+            "ddtrace/contrib/pymysql/*",
+        ],
+        "pylibmc": [
+            "ddtrace/contrib/pylibmc/*",
+        ],
+        "pytest": [
+            "ddtrace/contrib/pytest/*",
+        ],
+        "asynctest": [
+            "ddtrace/contrib/asynctest/*",
+        ],
+        "pytestbdd": [
+            "ddtrace/contrib/pytestbdd/*",
+        ],
+        "pymemcache": [
+            "ddtrace/contrib/pymemcache/*",
+        ],
+        "mongoengine": [
+            "ddtrace/contrib/mongoengine/*",
+        ],
+        "pymongo": [
+            "ddtrace/contrib/pymongo/*",
+        ],
+        "pynamodb": [
+            "ddtrace/contrib/pynamodb/*",
+        ],
+        "pyodbc": [
+            "ddtrace/contrib/pyodbc/*",
+        ],
+        "pyramid": [
+            "ddtrace/contrib/pyramid/*",
+        ],
+        "requests": [
+            "ddtrace/contrib/requests/*",
+        ],
+        "sanic": [
+            "ddtrace/contrib/sanic/*",
+        ],
+        "snowflake": [
+            "ddtrace/contrib/snowflake/*",
+        ],
+        "starlette": [
+            "ddtrace/contrib/starlette/*",
+        ],
+        "sqlalchemy": [
+            "ddtrace/contrib/sqlalchemy/*",
+        ],
+        "psycopg": [
+            "ddtrace/contrib/psycopg/*",
+        ],
+        "psycopg2": [
+            "ddtrace/contrib/psycopg2/*",
+        ],
+        "aiobotocore": [
+            "ddtrace/contrib/aiobotocore/*",
+        ],
+        "aiomysql": [
+            "ddtrace/contrib/aiomysql/*",
+        ],
+        "aiopg": [
+            "ddtrace/contrib/aiopg/*",
+        ],
+        "aioredis": [
+            "ddtrace/contrib/aioredis/*",
+        ],
+        "aredis": [
+            "ddtrace/contrib/aredis/*",
+        ],
+        "yaaredis": [
+            "ddtrace/contrib/yaaredis/*",
+        ],
+        "redis": [
+            "ddtrace/contrib/redis/*",
+        ],
+        "rediscluster": [
+            "ddtrace/contrib/rediscluster/*",
+        ],
+        "rq": [
+            "ddtrace/contrib/rq/*",
+        ],
+        "urllib3": [
+            "ddtrace/contrib/urllib3/*",
+        ],
+        "vertica": [
+            "ddtrace/contrib/vertica/*",
+        ],
+        "wsgi": [
+            "ddtrace/contrib/wsgi/*",
+        ],
+        "kombu": [
+            "ddtrace/contrib/kombu/*",
+        ],
+        "jinja2": [
+            "ddtrace/contrib/jinja2/*",
+        ],
+        "mako": [
+            "ddtrace/contrib/mako/*",
+        ],
+        "algoliasearch": [
+            "ddtrace/contrib/algoliasearch/*",
+        ],
+        "langchain": [
+            "ddtrace/contrib/langchain/*",
+        ],
     },
     "suites": {
         "debugger": [
             "@debugging",
             "@bootstrap",
             "@core",
             "@tracing"
-        ]
+        ],
+        "kafka": [
+            "@bootstrap",
+            "@core",
+            "@tracing",
+            "@kafka"
```

Comment:
This approach is totally fine, but in cases where we can expect to have a single pattern, like for most integrations, perhaps we can just put that pattern directly here instead of creating a dedicated component. WDYT?